x-pack/plugins/infra/public/components/logging/log_analysis_setup/setup_flyout/module_list_card.tsx

```diff
@@ -4,26 +4,41 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { EuiCard, EuiIcon } from '@elastic/eui';
+import { EuiCard, EuiIcon, EuiButtonEmpty } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
 import React from 'react';
 import { SetupStatus } from '../../../../../common/log_analysis';
 import { CreateJobButton, RecreateJobButton } from '../../log_analysis_setup/create_job_button';
+import { useLinkProps } from '../../../../hooks/use_link_props';
 
 export const LogAnalysisModuleListCard: React.FC<{
+  jobId: string;
   hasSetupCapabilities: boolean;
   moduleDescription: string;
   moduleName: string;
   moduleStatus: SetupStatus;
   onViewSetup: () => void;
-}> = ({ hasSetupCapabilities, moduleDescription, moduleName, moduleStatus, onViewSetup }) => {
+}> = ({
+  jobId,
+  hasSetupCapabilities,
+  moduleDescription,
+  moduleName,
+  moduleStatus,
+  onViewSetup,
+}) => {
   const moduleIcon =
     moduleStatus.type === 'required' ? (
       <EuiIcon size="xxl" type="machineLearningApp" />
     ) : (
       <EuiIcon color="secondary" size="xxl" type="check" />
     );
 
+  const viewInMlLinkProps = useLinkProps({
+    app: 'ml',
+    hash: '/jobs',
```

Comment:
Should use `pathname` instead of `hash` because this seems to be rewritten to `/apps/ml/jobs` immediately?

Reply:
Let me try :+1: